### PR TITLE
PS-5466: Release notes - v8.0.15-5

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

**Review by hrvojem — CHANGES_REQUESTED** [review]
We need to mention binlog encryption and that it is a breaking change.

**Comment by borys-belinsky** [comment]
Ready for review. At the point of the include directive, the release notes will expand to contain the following before rendering:
Starting from release :rn:`8.0.15-5`, |Percona Server| uses the
upstream implementation of binary log encryption. The variable
|encrypt-binlog| is removed and the related command line option
|opt.encrypt-binlog| is not supported. It is important that you remove
the |encrypt-binlog| variable from your configuration file before you
attempt to upgrade either from another release in the |Percona Server|
|version| series or from |Percona Server| 5.7.

.. seealso::

   |MySQL| Documentation - `Encrypting Binary Log Files and Relay Log
   Files
   <https://dev.mysql.com/doc/refman/8.0/en/replication-binlog-encryption.html>`_
   - `binlog_encryption variable
   <https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_encryption>`_

.. |encrypt-binlog| replace:: ``encrypt_binlog``
.. |opt.encrypt-binlog| replace:: ``--encrypt_binlog``

**Comment by george-lorch** [comment]
There should be a 'dummy' 8.0.14 release note that explains that we skipped this release number.

**Comment by borys-belinsky** [comment]
Updated and ready for review

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0. In previous releases, the audit log used to produce time stamps
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-5.rst]: Consider linking to feature docs

Is this prominent enough as it goes to the historically "blah blah" section of the release notes? Does it need an own section of "incompatible changes?"

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  had to be rolled back. Bug fixed :psbug:`5219`.
+
+Other bugs fixed:
+:psbug:`4106`,
+:psbug:`4107`,
+:psbug:`4108`,
+:psbug:`4121`,
+:psbug:`4474`,
+:psbug:`4640`,
+:psbug:`4996`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-5.rst]: Add this to PS-3956 above

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+:psbug:`5055`,
+:psbug:`5218`,
+:psbug:`5263`,
+:psbug:`5305`,
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-5.rst]: A stack buffer overrun could happen if redo log encryption with key rotation was enabled

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+:psbug:`5328`,
+:psbug:`5369`,
+:psbug:`5406`
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-5.rst]: If that was visible in release builds, then it's a very much visible item to document. @georgelorchpercona ?

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+.. |release| replace:: 8.0.15-5
+.. |date| replace:: March 11, 2019
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-5.rst]: Now it's in the past

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.15-5.rst (hunk 1 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-8.0.15-5.rst]: Ready for review: added a note like section titled "Incompatible changes"

**Review by laurynas-biveinis — APPROVED** [review]
OK but @georgelorchpercona still has to look at 5406 question

**Review by hrvojem — APPROVED** [review]

**Pull request merged**